tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -634,29 +543,9 @@ Update Seven filters footnote
     user clicks button  Save footnote
     user waits until page contains testid  footnote Updating ${SUBJECT_NAME} footnote
 
-Go to "Sign off" page for amendment
-    [Tags]  HappyPath
-    user clicks link   Sign off
```

Comment:
`user approves release for immediate publication` doesn't navigate to the Sign off page, so I think this is going to fail currently